basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/BaseLTIService.java

```diff
@@ -569,20 +569,28 @@ public List<Map<String, Object>> getToolsLaunch(String siteId) {
 		return getToolsLaunch(siteId, false);
 	}
 
+	@Override
+	public List<Map<String, Object>> getToolsLaunchCourseNav(String siteId, boolean includeStealthed) {
+		String query = "( lti_tools."+LTIService.LTI_MT_LAUNCH+" = 1 AND " +
+			"lti_tools."+LTIService.LTI_PL_COURSENAV+" = 1 )";
+		return getTools(query, null, 0, 0, siteId, includeStealthed);
```

Comment:
@ern @csev this is where the end-user needs to know they need to check the secret box "The tool URL can receive an LTI Resource Link launch" 😭 right?